.codecov.yml

```diff
@@ -3,6 +3,9 @@
 
 fixes:
   - "github.com/snapcore/snapd/::"
+  # default clone path on runner is /home/runner/work/snapd/snapd, drop the
+  # prefix so that C coverage data can match
+  - "/home/runner/work/snapd/snapd/::"
```

Comment:
Could we generate this file so that it uses an environment variable that knows the actual location.

Reply:
What you're not seeing here, is that codecov includes the prefix /home/runner/work in its list of hardcoded default path fixes. So while we technically could generate this on the fly, I'm not sure it's worth it